generators/server/templates/src/main/java/package/gateway/accesscontrol/_AccessControlFilter.java

```diff
@@ -55,7 +55,9 @@ public boolean shouldFilter() {
             if (requestUri.startsWith(serviceUrl.substring(0, serviceUrl.length() - 2))) {
                 if (isAuthorizedRequest(serviceUrl, serviceName, requestUri)) {
                     return false;
-                }
+                }else{
```

Comment:
In that case, instead of doing `if (isAuthorizedRequest(serviceUrl, serviceName, requestUri)) { return false; } else { return true; }` prefer doing ` return !isAuthorizedRequest(serviceUrl, serviceName, requestUri);`